### PR TITLE
Update MusicControlModule.java

### PR DESCRIPTION
**Pull request by bhram**
fix this issue https://github.com/tanguyantoine/react-native-music-control/issues/159

### CODE REVIEW

File: android/src/main/java/com/tanguyantoine/react/MusicControlModule.java [hunk 1]
```diff
@@ -238,7 +238,7 @@ synchronized public void setNowPlaying(ReadableMap metadata) {
         }
 
         nb.setContentTitle(title);
-        nb.setContentText(artist);
+        nb.setContentText(artist); ContextCompat.startForegroundService(context, new Intent(context, MusicControlNotification.NotificationService.class));
```
Comment (renatoalencar) [inline android/src/main/java/com/tanguyantoine/react/MusicControlModule.java]: I think this would lead to the same issue described before

Comment (bhram) [reply android/src/main/java/com/tanguyantoine/react/MusicControlModule.java]: it's work fine for me with no issues

**Comment by tanguyantoine** [comment]
@Guichaguri you are the expert :-)

File: android/src/main/java/com/tanguyantoine/react/MusicControlModule.java (hunk 1 above)
Comment (renatoalencar) [reply android/src/main/java/com/tanguyantoine/react/MusicControlModule.java]: For Android Oreo too?

Comment (bhram) [reply android/src/main/java/com/tanguyantoine/react/MusicControlModule.java]: yep

Comment (Guichaguri) [reply android/src/main/java/com/tanguyantoine/react/MusicControlModule.java]: Can you explain why do you need to add this here?

**Comment by Guichaguri** [comment]
Superseded by #162

**Pull request closed without merging**